photo_points/Controllers/AdminController.cs

```diff
@@ -17,32 +17,28 @@ namespace photo_points.Controllers
     public class AdminController : Controller
     {
 
-        private IAdminReviewServices _adminReviewServices;
-        private PhotoDataContext _dbc;
-
-        public AdminController(IAdminReviewServices adminServiceReview, PhotoDataContext dbc)
```

Comment:
The second part of the old line 23 "public AdminController(IAdminReviewServices adminServiceReview**, PhotoDataContext dbc**)" being removed seems to cause an issue with changing pages to pages that use the database

Reply:
Yah tested on my machine and replacing the new line 22 with this should fix it
`AdminController(IAdminReviewRepository adminReviewRepo, PhotoDataContext dbc)
        {
            _adminReviewRepo = adminReviewRepo;
            _dbc = dbc;
        }`